.travis.yml

```diff
@@ -0,0 +1,9 @@
+sudo: false
+language: ruby
+cache:
+  directories:
+    - vendor/bundle
+    - .gradle
+rvm:
+  - jruby-1.7.24
+script: rake bootstrap test:install-default && rake test:core test:plugins
```

Comment:
Easier if you use the scripts under the ```ci``` directory, so in case we change the tasks names (for example between versions) we don't have to update the set config.